x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -402,6 +409,7 @@ export interface XYArgs {
 export interface XYState {
   preferredSeriesType: SeriesType;
   legend: LegendConfig;
+  valueLabels: ValueLabelConfig;
```

Comment:
As this is not optional, it changes the current state shape and we require a migration to make it work (existing saved objects won't have the `valueLabels` property. Let's either make it optional (and default the behavior to `hide`) or add the migration using the regular migration framework